acl/acl.d.ts

```diff
@@ -3,7 +3,7 @@
 // Definitions by: Qubo <https://github.com/tkQubo>
 // Definitions: https://github.com/borisyankov/DefinitelyTyped
 
-/// <reference path="../bluebird/bluebird.d.ts" />
+/// <reference path='../es6-promise/es6-promise.d.ts' />
```

Comment:
You are right. I changed it because Travis gave an error about a double definition of Promise, since now mongodb uses es6-promise.
Do you know how can I avoid the error when reverting to bluebird in acl?
Thanks